content/en/docs/device-provisioning/self-update/self-update-tutorial.md

```diff
@@ -34,6 +34,182 @@ This chapter describes the steps necessary to perform a local (without cloud) se
 
 ## Self-Update Trigger Message
 
-`start-update-example.yaml` file:
-
-{{<readfile file="start-update-example.yaml" code="true" lang="yaml">}}
+`start-update-example.json` file:
+
+{{<readfile file="start-update-example.json" code="true" lang="json">}}
+
+## Example Message Flows
+
+### Current State
+
+1. Initial response message on startup from self update agent in topic `selfupdate/currentstate`, or upon request by sending message to `selfupdate/currentstate/get`
+
+      Request:
+      ```json
+      {
+            "activityId": "<uuid>",
+            "timestamp": 1676332092
+      }
+      ```
+
+      Response:
+      ```json
+      {
+            "activityId": "<uuid>",
+            "timestamp": 1675345910,
+            "payload": {
+                  "domains": [
+                        {
+                              "id": "self-update",
+                              "components": [
+                              {
+                                    "id": "os-image",
+                                    "version": "bundle_version_not_available"
+                              }
+                              ]
+                        }
+                  ]
+            }
+      }
+      ```
+
+
+### Desired State
+
+1. External trigger to update via desired state on topic `selfupdate/desiredstate`:
+
+      ```json
+      {
+            "activityId": "random-uuid-as-string",
```

Comment:
Is it like previous "activityId": "<uuid>",

Reply:
Fixed